### ISSUE TITLE
Add timezone variable

### ISSUE THREAD

**Issue by danbohea** [hashbangcode/vlad#236]
We should add a timezone variable so that logs, archive dumps, db dumps etc all make a bit more sense (mine are all out by an hour here in the UK).

**Comment by philipnorton42** [comment]
Would this just be for the php.ini file? This already exists in the form of 'php_date_timezone'.

**Comment by danbohea** [comment]
Not sure. I meant setting the timezone for the server but I'll settle for anything that does the job.

**Comment by danbohea** [comment]
According to the docs, the default value of `php_date_timezone` is `Europe/London` - you'd think I wouldn't have noticed an issue if this was sufficient.

**Comment by philipnorton42** [comment]
No, I see what you mean, and php_date_timezone doesn't cover it. I think we just need to look into NTP services a little and that should solve the issue here.

**Comment by danbohea** [comment]
I made a start on reading up on this but got sidetracked by project work.

- Show timezone in Ubuntu: http://askubuntu.com/a/502082/219377
- Set timezone in Ubuntu: http://askubuntu.com/a/594186/219377

**Comment by iAugur** [comment]
I have a role I use for that if you are interested?
https://github.com/Blue-Bag/ansible-role-first5ntp
Bit Debian biased - but could be updated

**Comment by philipnorton42** [comment]
Hi George, That's a good idea. Worth looking into :)
